src/test/files/java/ConfigFriendlyNameCheckMultipleConfig.java

```diff
@@ -0,0 +1,25 @@
+import org.mule.api.annotations.components.MetaDataCategory;
+
+import org.mule.api.annotations.Connector;
+
```

Comment:
Let's add the imports for all the annotations we're using here so that the testing framework can pick them up (and not log the warnings).